composer.json

```diff
@@ -38,11 +38,15 @@
         "phpstan/phpstan-phpunit":      "^0.10",
         "phpstan/phpstan-strict-rules": "^0.10.1",
         "squizlabs/php_codesniffer":    "^3.3.1",
-        "doctrine/coding-standard":     "^4.0.0"
+        "doctrine/coding-standard":     "^4.0.0",
+        "zendframework/zend-expressive-session": "^1.2.1"
```

Comment:
I wonder if having this as dev-dependency is enough. Thoughts?

Reply:
Until yet no existing code was changed, once this would happen, i suggest to create a new repository as you suggested.

Reply:
The alternative is to make sure that `<1.2,>2.0` are in `"conflict"`

Reply:
good point, doing the same on my libs

Reply:
done